third_party/move/move-model/bytecode/src/fat_loop.rs

```diff
@@ -476,21 +476,20 @@ impl FatLoopBuilder {
             if cfg.successors(*block_id).len() == 1 && !block[block.len() - 1].is_branching() {
                 let successor_id = cfg.successors(*block_id).first().unwrap();
                 let successor_label_opt = id_label_map.get(successor_id).copied();
-                if let Some(successor_label) = successor_label_opt
```

Comment:
Seems like old Rust formatting, these should be rolled back?

Reply:
This should be irrelevant to Move version though? New Rust (1..89) allows this and codebase is now using 1.89